### PR TITLE
Adding amberelectric documentation

### PR DESCRIPTION
**Pull request by madpilot**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/56448
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/amberelectric.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Amber Electric
+description: Instructions on how to integrate Amber Electric live prices within Home Assistant.
+ha_category:
+  - Sensor
```
Comment (frenck) [inline source/_integrations/amberelectric.markdown:5]: Energy?

File: source/_integrations/amberelectric.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - "@madpilot"
+ha_domain: amberelectric
+ha_platforms:
+  - sensor
+---
+
+[Amber](https://www.amber.com.au/) is an Australian electricity retailer that provides access to wholesale electricity prices. Customers monitor the wholesale price and shift their energy usage to cheaper, greener times. This saves them money and supports the shift to a more renewably-powered Australia. Using the `amberelectric` component, customers can go a step further - setting up devices to automatically shift energy usage to cheaper and greener times, based on real-time electricity prices and forecasts”
```
Comment (frenck) [inline source/_integrations/amberelectric.markdown]:
```suggestion
[Amber](https://www.amber.com.au/) is an Australian electricity retailer that provides access to wholesale electricity prices. Customers monitor the wholesale price and shift their energy usage to cheaper, greener times. This saves them money and supports the shift to a more renewably-powered Australia.

Using this integration, customers can go a step further - setting up devices to automatically shift energy usage to cheaper and greener times, based on real-time electricity prices and forecasts.
```

**Review by frenck — CHANGES_REQUESTED** [review]
The installation instructions for Home Assistant are missing on this page. I would suggest (`{% include integrations/config_flow.md %}`.

Additionally, I would recommend putting up the prerequisites (API key) and installation instructions directly after the introduction and move additional documentation below that.

File: source/_integrations/amberelectric.markdown (hunk 2 above)
Comment (madpilot) [reply source/_integrations/amberelectric.markdown]: Good feedback - paragraph is now split

**Review by frenck — APPROVED** [review]
Thanks, @madpilot 👍

**Pull request merged**